tox.ini

```diff
@@ -6,9 +6,10 @@ envlist=
 [isort]
 combine_as_imports=True
 force_sort_within_sections=True
+force_grid_wrap=1
 known_third_party=hypothesis,pytest
-profile=black
```

Comment:
nit: Is moving this necessary? might just messy up some commit history. I'm fine with it though.

Reply:
Ha, nope. Not sure why that got moved 🤔